schema/index.graphql

```diff
@@ -171,16 +281,19 @@ A warehouse
 type Warehouse {
   id: ID!
   name: String!
-  openingTimes: [OpeningTime!]!
+  openingTimes: [DateTime!]!
```

Comment:
Only the opening times for a warehouse are not enough to display a datetime picker for example for click and collect, as the opening times can be interrupted for example by bank holidays.